iis-wf/iis-wf-export-actionmanager/src/test/resources/eu/dnetlib/iis/wf/export/actionmanager/entity/patent/default/output/patent1.properties

```diff
@@ -76,6 +76,11 @@ payload.instance[0].hostedby.key=10|openaire____::79e8e56d43f34eb7698bfb7535d1b3
 payload.instance[0].hostedby.value=European Patent Office/PATSTAT
 payload.instance[0].collectedfrom.key=10|openaire____::79e8e56d43f34eb7698bfb7535d1b37a
 payload.instance[0].collectedfrom.value=European Patent Office/PATSTAT
+payload.instance[0].pid[0].value=EPpatent1
+payload.instance[0].pid[0].qualifier.classid=epo_id
+payload.instance[0].pid[0].qualifier.classname=epo_id
+payload.instance[0].pid[0].qualifier.schemeid=dnet:pid_types
+payload.instance[0].pid[0].qualifier.schemename=dnet:pid_types
```

Comment:
Shouldn't we also test `payload.instance[0].pid[1]` (which was originally defined as `payload.pid[1]`) in the very same way?

Reply:
Argh, missed that second pid:/ Adding tests.